developer/src/kmc-package/test/package-compiler.tests.ts

```diff
@@ -86,7 +86,7 @@ describe('KmpCompiler', function () {
               assert.deepEqual(kmpJsonOutput, kmpJsonZippedFixture);
             }),
             zipFile.file(`${modelID}.model.js`).async('uint8array').then(modelJsFile => {
-              assert.deepEqual(modelJsFile, fs.readFileSync(jsPath));
+              assert.deepEqual(modelJsFile, new Uint8Array(fs.readFileSync(jsPath)));
```

Comment:
recent ts version complains about this at compile because Buffer not strictly equivalent to Uint8Array.